python/cudf/udf_cpp/CMakeLists.txt

```diff
@@ -19,29 +19,29 @@ include(rapids-cpm)
 include(rapids-cuda)
 include(rapids-find)
 
-rapids_cuda_init_architectures(strings-udf-cpp)
+rapids_cuda_init_architectures(udf-cpp)
```

Comment:
You should now be able to get rid of
- Any rapids-cuda
- The `project`

Reply:
made a few cmake updates in https://github.com/rapidsai/cudf/pull/12669/commits/9b175e90eaac3c581a98de179bd91a428ab7b961, there's also some updates I needed to make to how things are `include`ed in that commit, but I think they're unrelated.